day_6/exercises/burrito.js

```diff
@@ -16,12 +16,34 @@ class Burrito {
     this.base = base;
     this.toppings = toppings;
   }
+
+  addTopping(topping) {
+    this.toppings.push(topping);
+  }
+
+  removeTopping(topping) {
+    for (var i = 0; i < this.toppings.length; i++){
+      if (this.toppings[i]==topping){
+        this.toppings.splice(i,1);
```

Comment:
Nice job iterating through your toppings to find one that matches the topping to remove!